### DISCUSSION TITLE
endereço da API

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#493] [Tira Dúvidas] [answered]
Balta 
estou fazendo o curso de ASP.NET 6 (API), em meus testes a a Api só roda em http não roda (nem aparece) https, isso é algum tipo de configuração?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FBODeveloper, como vai?

Tenta executar os comandos para gerar novamente os certificados HTTPS:
```
dotnet dev-certs https --clean
dotnet dev-certs https --trust
```

Feito isso, reinicia o VS (PC se possível) e testa novamente por favor.

**Reply by FBODeveloper** [reply]
Bom dia, Balta....
Cara ainda continua não gerando
![image](https://github.com/balta-io/forum/assets/146769790/0e56216d-947e-4287-a963-14a5dd085561)

**Reply by andrebaltieri** [reply]
Isto mesmo criando um novo projeto, do zero, com .NET 6+?

**Reply by FBODeveloper** [reply]
Deu certo valeu

**Comment by FBODeveloper** [comment]
Isso não vou tentar...
Porém o mesmo projeto no visual Studio roda HTTPS normal
Vou criar um projeto novo no vscode e testar valeu fera

Em sáb., 17 de fev. de 2024 11:20, Andre Baltieri ***@***.***>
escreveu:
